pages/security/human-security/passwords/fr.md

```diff
@@ -0,0 +1,64 @@
+Il y a trois sortes de gestionnaire de mots de passe dont il faut être conscient.
+
+* **Application locale:** C'est un logiciel local qui entrepose tous les mots de passe sur votre ordinateur local, chiffrés avec le mots de passe maitre.  C'est le plus sûre parmis les trois options, mais il peut être difficile de synchroniser vos mots de passe entre de multiples appareils.
+* **Service infonuagique (cloud service):** C'est un service, normalement payant, qui entrepose vos mots de  passe, peu importe l'appareil.  Ce service vous donne accès à vos mots de passe peu  importe où vous soyez.  Le prix à payer pour cet avantage est un niveau de sécurité inférieur à une application locale.
+* **Extension de navigateur web:** Souvent et des applications locales et des services infonuagiques offrent des extensions pour les navigateur web permettant d'accéder aux mots de passe directement depuis le navigateur.  C'est pratique, mais au prix d'une sécurité légèrement affaiblie.
+
+Peu importe l'option que vous choisissez, la clé est d'utiliser un gestionnaire de mots de passe.  Voici 2 astuces à garder en tête:
+
+* **Mot de passe maître:**  Lorsqu'on se sert d'un gestionnaire de mots de passe il est essentiel, primordial, de ne pas perdre le mot de passe maître.  Sans le mot de passe maître tous les autres mots de passe sont inaccessible.  Si vous pensez qu'il est possible d'oublier ce mot de passe maître, écrivez-le à quelque part et garder le bout de papier dans un lieu sûr.
+* **Copie de sauvegarde (backups):** Il est très important de faire des copies de sauvegarde de votre base de données des mots de passe.  Dans le cas des services infonuagiques, les sauvegardes sont  faits pour vous (il est quand même bien de faire une copie locale).  Dans le  cas des applications locales il suffit de faire une copie du fichier de données.
+
+Gestionnaires de mots de passe populaires:
+
+* KeePass et KeePassX (applications) sont 2 versions d'un même gestionnaire de mots de passe.  Les 2 sont très bien cotés.  Ces 2 outils utilisent le même format de fichier.  Ils roulent sur presque tous les ordinateurs.
+   * [Surveillance Self-defense / Guide pratique: utiliser KeePassXC](https://ssd.eff.org/fr/module/guide-pratique-utiliser-keepassxc)
+   * [KeePassX for Windows / Secure Password Manager](https://securityinabox.org/fr/guide/keepassx/windows)
+* LastPass (infonuagique)
+* 1Password (infonuagique et application)
+
+# Utiliser des mots de passe robustes
+
+Les mots de passe forts sont *générés de manière aléatoire*.  Mise-à-part les mots de passe pour votre appareil et de votre gestionnaire de mots de passe, tous vos mots de passe devraient être générés par le gestionnaire et devraient être d'une longueur minimale de 12 caractères.  Les mots de passe n'ont pas besoin d'être plus longs que 26 caractères.
+
+Les être humains ne créent pas de mots de passe robustes, mais les ordinateurs, eux, en créent d'excellents.  Laissez votre ordinateur générer vos mots de passe.
```

Comment:
* In "Les être humains", "être" takes an "s" at the end.

Reply:
fixed